Wikipedia/Code/WMFAuthenticationManager.swift

```diff
@@ -183,19 +168,32 @@ public class WMFAuthenticationManager: NSObject {
     /**
      *  Logs out any authenticated user and clears out any associated cookies
      */
-    @objc public func logout(completion: @escaping () -> Void = {}){
-        logoutManager = AFHTTPSessionManager(baseURL: loginSiteURL)
-        _ = logoutManager?.wmf_apiPOST(with: ["action": "logout", "format": "json"], success: { (_, response) in
-            DDLogDebug("Successfully logged out, deleted login tokens and other browser cookies")
-            // It's best to call "action=logout" API *before* clearing local login settings...
-            self.resetLocalUserLoginSettings()
-            completion()
-        }, failure: { (_, error) in
-            // ...but if "action=logout" fails we *still* want to clear local login settings, which still effectively logs the user out.
-            DDLogDebug("Failed to log out, delete login tokens and other browser cookies: \(error)")
+    @objc public func logout(completion: @escaping () -> Void = {}) {
+        let reset = {
+            DDLogDebug("Deleted login tokens and other browser cookies")
             self.resetLocalUserLoginSettings()
+        }
+        let taskGroup = WMFTaskGroup()
+        for loginSite in LoginSite.allCases {
```

Comment:
On the web, it looks like logging out of either wikidata or any centralauth wikipedia site logs the user out of all projects - is there something else causing the stored cookies to stick around in the app that requires logging out of both sites?

Reply:
good point, I assumed that I had to do it for each site, will fix 🙌